CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+###Standard Question Setup###
+def multich(ch1="error1", ch2="error2", ch3="error3", ch4="error4"):
+	choice= input(f"""
+	Select Option:
+		A. {ch1}
+		B. {ch2}
+		C. {ch3}
+		D. {ch4}""")
+	return choice
+	
+###Questions for naming and selecting play class.###
+def givename():
+	name= input(print("Type your name."))
+	return name
+def giveclass():
+	playclass= multich(ch1='Warrior', ch2='Wizard', ch3='Cleric', ch4='Rogue')
+	return playclass
+	#add details about stats here
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+
+###Define character class###
+class Character:
+	def __init__(self, name, position, items):
+		self.name= name
+		self.position= position
+		self.items= items
+#Setters and Getters for all Characters
+#Name		
+	def name(self):
+		return self.name
+	def name(self, value):
```

Comment:
I don't think this does what you think it does anyway.

Reply:
Clearly. I dont know why I would want it to be a property instead of an open field.